### ISSUE TITLE
v5.1.0 - Op.regexp add unwanted backslahes & double quotes that breaks regex.

### ISSUE THREAD

**Issue by Sorok-Dva** [sequelize/sequelize#10602]
## What are you doing?
I try to use a regex to match one or many terms in the column. 
```js

let services = ['First'];

Model.findAll({
  where: {
    column: {
      [Op.regexp]: services[0]
    }
  }
});
```

## What do you expect to happen?
I want that the mysql request look like that : 

```
 `Model`.`column` REGEXP 'First'
```

## What is actually happening?
Backslahes and doubles quotes are added in the generated request.

```
`Model`.`column` REGEXP  '\"First\"'
```

__Dialect:__ mysql
__Dialect version:__ NA
__Database version:__ 5.7.25-0ubuntu0.18.04.2
__Sequelize version:__ 5.1.0
__Tested with latest release:__ Yes (5.1.0)

**Comment by SimonSchick** [comment]
Please provide a full repro, we have a bunch of tests that specifically cover this functionality and they are all passing.

**Comment by Sorok-Dva** [comment]
Hi, here is the complete functional code.
```javascript
let { filterQuery } = req.body;
let query = {
  where: { es_id: filterQuery.establishments },
  attributes: { exclude: ['lat', 'lon'] },
  group: ['Wish->Candidate.id'],
  include: [{
    model: Models.Wish,
    required: true,
    on: {
      '$Application.wish_id$': {
        [Op.col]: 'Wish.id'
      }
    },
    where: {
      [Op.col]: Sequelize.where(Sequelize.fn('lower', Sequelize.col('posts')), {
        [Op.like]: `%${req.body.post.toLowerCase()}%`
      })
    },
    include: {
      model: Models.Candidate,
      attributes: { exclude: ['updatedAt', 'createdAt'] },
      required: true,
      include: [{
        model: Models.User,
        attributes: { exclude: ['password', 'type', 'role', 'email', 'phone', 'updatedAt', 'createdAt'] },
        on: {
          '$Wish->Candidate.user_id$': {
            [Op.col]: 'Wish->Candidate->User.id'
          }
        },
        required: true
      }, {
        model: Models.Experience,
        as: 'experiences',
      }, {
        model: Models.CandidateDocument,
        as: 'documents',
        attributes: ['candidate_id', 'type'],
      }, {
        model: Models.CandidateFormation,
        as: 'formations',
      }]
    }
  }, {
    model: Models.Establishment,
    attributes: ['id'],
    on: {
      '$Application.es_id$': {
        [Op.col]: 'Establishment.id'
      }
    },
    include: [{
      model: Models.FavoriteCandidate,
      attributes: ['added_by', 'candidate_id'],
      on: {
        '$Establishment.id$': {
          [Op.col]: 'Establishment->FavoriteCandidates.es_id'
        },
        '$Wish->Candidate.id$': {
          [Op.col]: 'Establishment->FavoriteCandidates.candidate_id'
        }
      },
      include: {
        model: Models.User,
        attributes: ['firstName', 'lastName'],
        on: {
          '$Establishment->FavoriteCandidates.added_by$': {
            [Op.col]: 'Establishment->FavoriteCandidates->User.id'
          }
        }
      }
    }, {
      model: Models.ArchivedCandidate,
      attributes: ['added_by', 'candidate_id'],
      on: {
        '$Establishment.id$': {
          [Op.col]: 'Establishment->ArchivedCandidates.es_id'
        },
        '$Wish->Candidate.id$': {
          [Op.col]: 'Establishment->ArchivedCandidates.candidate_id'
        }
      },
    }]
  }]
};
if (!_.isNil(filterQuery.contractType)) query.include[0].where.contract_type = filterQuery.contractType;
if (!_.isNil(filterQuery.services)) {
  query.include[0].where.services = {
    [Op.regexp]: `(${filterQuery.services[0]})`
  };
  console.log(query.include[0].where.services);
}
Models.Application.findAll(query).then(applications => {
  return res.status(200).send(applications);
}).catch(error => next(new BackError(error)));
```

The generated request is : 

```
SELECT `Application`.`id`, `Application`.`name`, `Application`.`wish_id`, `Application`.`candidate_id`, `Application`.`es_id`, `Application`.`ref_es_id`, `Application`.`percentage`, `Application`.`accepted`, `Application`.`status`, `Application
`.`type`, `Application`.`new`, `Application`.`renewed`, `Application`.`last_renew`, `Application`.`createdAt`, `Application`.`updatedAt`, `Wish`.`id` AS `Wish.id`, `Wish`.`candidate_id` AS `Wish.candidate_id`, `Wish`.`name` AS `Wish.name`, `Wish`.`contract_type` AS
 `Wish.contract_type`, `Wish`.`posts` AS `Wish.posts`, `Wish`.`services` AS `Wish.services`, `Wish`.`full_time` AS `Wish.full_time`, `Wish`.`part_time` AS `Wish.part_time`, `Wish`.`day_time` AS `Wish.day_time`, `Wish`.`night_time` AS `Wish.night_time`, `Wish`.`libe
ral_cabinets` AS `Wish.liberal_cabinets`, `Wish`.`availability` AS `Wish.availability`, `Wish`.`start` AS `Wish.start`, `Wish`.`end` AS `Wish.end`, `Wish`.`status` AS `Wish.status`, `Wish`.`lat` AS `Wish.lat`, `Wish`.`lon` AS `Wish.lon`, `Wish`.`geolocation` AS `Wi
sh.geolocation`, `Wish`.`custom_address` AS `Wish.custom_address`, `Wish`.`es_count` AS `Wish.es_count`, `Wish`.`createdAt` AS `Wish.createdAt`, `Wish`.`updatedAt` AS `Wish.updatedAt`, `Wish->Candidate`.`user_id` AS `Wish.Candidate.user_id`, `Wish->Candidate`.`es_i
d` AS `Wish.Candidate.es_id`, `Wish->Candidate`.`description` AS `Wish.Candidate.description`, `Wish->Candidate`.`photo` AS `Wish.Candidate.photo`, `Wish->Candidate`.`video` AS `Wish.Candidate.video`, `Wish->Candidate`.`status` AS `Wish.Candidate.status`, `Wish->Ca
ndidate`.`note` AS `Wish.Candidate.note`, `Wish->Candidate`.`views` AS `Wish.Candidate.views`, `Wish->Candidate`.`is_available` AS `Wish.Candidate.is_available`, `Wish->Candidate`.`id` AS `Wish.Candidate.id`, `Wish->Candidate->User`.`id` AS `Wish.Candidate.User.id`
, `Wish->Candidate->User`.`firstName` AS `Wish.Candidate.User.firstName`, `Wish->Candidate->User`.`lastName` AS `Wish.Candidate.User.lastName`, `Wish->Candidate->User`.`birthday` AS `Wish.Candidate.User.birthday`, `Wish->Candidate->User`.`postal_code` AS `Wish.Cand
idate.User.postal_code`, `Wish->Candidate->User`.`town` AS `Wish.Candidate.User.town`, `Wish->Candidate->User`.`key` AS `Wish.Candidate.User.key`, `Wish->Candidate->User`.`validated` AS `Wish.Candidate.User.validated`, `Wish->Candidate->experiences`.`id` AS `Wish.C
andidate.experiences.id`, `Wish->Candidate->experiences`.`name` AS `Wish.Candidate.experiences.name`, `Wish->Candidate->experiences`.`candidate_id` AS `Wish.Candidate.experiences.candidate_id`, `Wish->Candidate->experiences`.`poste_id` AS `Wish.Candidate.experience
s.poste_id`, `Wish->Candidate->experiences`.`service_id` AS `Wish.Candidate.experiences.service_id`, `Wish->Candidate->experiences`.`internship` AS `Wish.Candidate.experiences.internship`, `Wish->Candidate->experiences`.`start` AS `Wish.Candidate.experiences.start`
, `Wish->Candidate->experiences`.`end` AS `Wish.Candidate.experiences.end`, `Wish->Candidate->experiences`.`current` AS `Wish.Candidate.experiences.current`, `Wish->Candidate->experiences`.`createdAt` AS `Wish.Candidate.experiences.createdAt`, `Wish->Candidate->exp
eriences`.`updatedAt` AS `Wish.Candidate.experiences.updatedAt`, `Wish->Candidate->documents`.`id` AS `Wish.Candidate.documents.id`, `Wish->Candidate->documents`.`candidate_id` AS `Wish.Candidate.documents.candidate_id`, `Wish->Candidate->documents`.`type` AS `Wish
.Candidate.documents.type`, `Wish->Candidate->formations`.`id` AS `Wish.Candidate.formations.id`, `Wish->Candidate->formations`.`candidate_id` AS `Wish.Candidate.formations.candidate_id`, `Wish->Candidate->formations`.`name` AS `Wish.Candidate.formations.name`, `Wi
sh->Candidate->formations`.`start` AS `Wish.Candidate.formations.start`, `Wish->Candidate->formations`.`end` AS `Wish.Candidate.formations.end`, `Wish->Candidate->formations`.`createdAt` AS `Wish.Candidate.formations.createdAt`, `Wish->Candidate->formations`.`updat
edAt` AS `Wish.Candidate.formations.updatedAt`, `Establishment`.`id` AS `Establishment.id`, `Establishment->FavoriteCandidates`.`id` AS `Establishment.FavoriteCandidates.id`, `Establishment->FavoriteCandidates`.`added_by` AS `Establishment.FavoriteCandidates.added_
by`, `Establishment->FavoriteCandidates`.`candidate_id` AS `Establishment.FavoriteCandidates.candidate_id`, `Establishment->FavoriteCandidates->User`.`id` AS `Establishment.FavoriteCandidates.User.id`, `Establishment->FavoriteCandidates->User`.`firstName` AS `Estab
lishment.FavoriteCandidates.User.firstName`, `Establishment->FavoriteCandidates->User`.`lastName` AS `Establishment.FavoriteCandidates.User.lastName`, `Establishment->ArchivedCandidates`.`id` AS `Establishment.ArchivedCandidates.id`, `Establishment->ArchivedCandida
tes`.`added_by` AS `Establishment.ArchivedCandidates.added_by`, `Establishment->ArchivedCandidates`.`candidate_id` AS `Establishment.ArchivedCandidates.candidate_id` FROM `Applications` AS `Application` INNER JOIN `Wishes` AS `Wish` ON `Application`.`wish_id` = `Wi
sh`.`id` AND lower(`posts`) LIKE '%asd%' AND `Wish`.`contract_type` = 'cdi-cdd' AND `Wish`.`services` REGEXP '\"(Radiothérapie)\"' INNER JOIN `Candidates` AS `Wish->Candidate` ON `Wish`.`candidate_id` = `Wish->Candidate`.`id` INNER JOIN `Users` AS `Wish->Candidate-
>User` ON `Wish->Candidate`.`user_id` = `Wish->Candidate->User`.`id` LEFT OUTER JOIN `Experiences` AS `Wish->Candidate->experiences` ON `Wish->Candidate`.`id` = `Wish->Candidate->experiences`.`candidate_id` LEFT OUTER JOIN `CandidateDocuments` AS `Wish->Candidate->
documents` ON `Wish->Candidate`.`id` = `Wish->Candidate->documents`.`candidate_id` LEFT OUTER JOIN `CandidateFormations` AS `Wish->Candidate->formations` ON `Wish->Candidate`.`id` = `Wish->Candidate->formations`.`candidate_id` LEFT OUTER JOIN `Establishments` AS `E
stablishment` ON `Application`.`es_id` = `Establishment`.`id` LEFT OUTER JOIN `FavoriteCandidates` AS `Establishment->FavoriteCandidates` ON `Establishment`.`id` = `Establishment->FavoriteCandidates`.`es_id` AND `Wish->Candidate`.`id` = `Establishment->FavoriteCand
idates`.`candidate_id` LEFT OUTER JOIN `Users` AS `Establishment->FavoriteCandidates->User` ON `Establishment->FavoriteCandidates`.`added_by` = `Establishment->FavoriteCandidates->User`.`id` LEFT OUTER JOIN `ArchivedCandidates` AS `Establishment->ArchivedCandidates
` ON `Establishment`.`id` = `Establishment->ArchivedCandidates`.`es_id` AND `Wish->Candidate`.`id` = `Establishment->ArchivedCandidates`.`candidate_id` WHERE `Application`.`es_id` IN ('5') GROUP BY `Wish->Candidate`.`id`;
```

Instead of : 
```
SELECT `Application`.`id`, `Application`.`name`, `Application`.`wish_id`, `Application`.`candidate_id`, `Application`.`es_id`, `Application`.`ref_es_id`, `Application`.`percentage`, `Application`.`accepted`, `Application`.`status`, `Application
`.`type`, `Application`.`new`, `Application`.`renewed`, `Application`.`last_renew`, `Application`.`createdAt`, `Application`.`updatedAt`, `Wish`.`id` AS `Wish.id`, `Wish`.`candidate_id` AS `Wish.candidate_id`, `Wish`.`name` AS `Wish.name`, `Wish`.`contract_type` AS
 `Wish.contract_type`, `Wish`.`posts` AS `Wish.posts`, `Wish`.`services` AS `Wish.services`, `Wish`.`full_time` AS `Wish.full_time`, `Wish`.`part_time` AS `Wish.part_time`, `Wish`.`day_time` AS `Wish.day_time`, `Wish`.`night_time` AS `Wish.night_time`, `Wish`.`libe
ral_cabinets` AS `Wish.liberal_cabinets`, `Wish`.`availability` AS `Wish.availability`, `Wish`.`start` AS `Wish.start`, `Wish`.`end` AS `Wish.end`, `Wish`.`status` AS `Wish.status`, `Wish`.`lat` AS `Wish.lat`, `Wish`.`lon` AS `Wish.lon`, `Wish`.`geolocation` AS `Wi
sh.geolocation`, `Wish`.`custom_address` AS `Wish.custom_address`, `Wish`.`es_count` AS `Wish.es_count`, `Wish`.`createdAt` AS `Wish.createdAt`, `Wish`.`updatedAt` AS `Wish.updatedAt`, `Wish->Candidate`.`user_id` AS `Wish.Candidate.user_id`, `Wish->Candidate`.`es_i
d` AS `Wish.Candidate.es_id`, `Wish->Candidate`.`description` AS `Wish.Candidate.description`, `Wish->Candidate`.`photo` AS `Wish.Candidate.photo`, `Wish->Candidate`.`video` AS `Wish.Candidate.video`, `Wish->Candidate`.`status` AS `Wish.Candidate.status`, `Wish->Ca
ndidate`.`note` AS `Wish.Candidate.note`, `Wish->Candidate`.`views` AS `Wish.Candidate.views`, `Wish->Candidate`.`is_available` AS `Wish.Candidate.is_available`, `Wish->Candidate`.`id` AS `Wish.Candidate.id`, `Wish->Candidate->User`.`id` AS `Wish.Candidate.User.id`
, `Wish->Candidate->User`.`firstName` AS `Wish.Candidate.User.firstName`, `Wish->Candidate->User`.`lastName` AS `Wish.Candidate.User.lastName`, `Wish->Candidate->User`.`birthday` AS `Wish.Candidate.User.birthday`, `Wish->Candidate->User`.`postal_code` AS `Wish.Cand
idate.User.postal_code`, `Wish->Candidate->User`.`town` AS `Wish.Candidate.User.town`, `Wish->Candidate->User`.`key` AS `Wish.Candidate.User.key`, `Wish->Candidate->User`.`validated` AS `Wish.Candidate.User.validated`, `Wish->Candidate->experiences`.`id` AS `Wish.C
andidate.experiences.id`, `Wish->Candidate->experiences`.`name` AS `Wish.Candidate.experiences.name`, `Wish->Candidate->experiences`.`candidate_id` AS `Wish.Candidate.experiences.candidate_id`, `Wish->Candidate->experiences`.`poste_id` AS `Wish.Candidate.experience
s.poste_id`, `Wish->Candidate->experiences`.`service_id` AS `Wish.Candidate.experiences.service_id`, `Wish->Candidate->experiences`.`internship` AS `Wish.Candidate.experiences.internship`, `Wish->Candidate->experiences`.`start` AS `Wish.Candidate.experiences.start`
, `Wish->Candidate->experiences`.`end` AS `Wish.Candidate.experiences.end`, `Wish->Candidate->experiences`.`current` AS `Wish.Candidate.experiences.current`, `Wish->Candidate->experiences`.`createdAt` AS `Wish.Candidate.experiences.createdAt`, `Wish->Candidate->exp
eriences`.`updatedAt` AS `Wish.Candidate.experiences.updatedAt`, `Wish->Candidate->documents`.`id` AS `Wish.Candidate.documents.id`, `Wish->Candidate->documents`.`candidate_id` AS `Wish.Candidate.documents.candidate_id`, `Wish->Candidate->documents`.`type` AS `Wish
.Candidate.documents.type`, `Wish->Candidate->formations`.`id` AS `Wish.Candidate.formations.id`, `Wish->Candidate->formations`.`candidate_id` AS `Wish.Candidate.formations.candidate_id`, `Wish->Candidate->formations`.`name` AS `Wish.Candidate.formations.name`, `Wi
sh->Candidate->formations`.`start` AS `Wish.Candidate.formations.start`, `Wish->Candidate->formations`.`end` AS `Wish.Candidate.formations.end`, `Wish->Candidate->formations`.`createdAt` AS `Wish.Candidate.formations.createdAt`, `Wish->Candidate->formations`.`updat
edAt` AS `Wish.Candidate.formations.updatedAt`, `Establishment`.`id` AS `Establishment.id`, `Establishment->FavoriteCandidates`.`id` AS `Establishment.FavoriteCandidates.id`, `Establishment->FavoriteCandidates`.`added_by` AS `Establishment.FavoriteCandidates.added_
by`, `Establishment->FavoriteCandidates`.`candidate_id` AS `Establishment.FavoriteCandidates.candidate_id`, `Establishment->FavoriteCandidates->User`.`id` AS `Establishment.FavoriteCandidates.User.id`, `Establishment->FavoriteCandidates->User`.`firstName` AS `Estab
lishment.FavoriteCandidates.User.firstName`, `Establishment->FavoriteCandidates->User`.`lastName` AS `Establishment.FavoriteCandidates.User.lastName`, `Establishment->ArchivedCandidates`.`id` AS `Establishment.ArchivedCandidates.id`, `Establishment->ArchivedCandida
tes`.`added_by` AS `Establishment.ArchivedCandidates.added_by`, `Establishment->ArchivedCandidates`.`candidate_id` AS `Establishment.ArchivedCandidates.candidate_id` FROM `Applications` AS `Application` INNER JOIN `Wishes` AS `Wish` ON `Application`.`wish_id` = `Wi
sh`.`id` AND lower(`posts`) LIKE '%asd%' AND `Wish`.`contract_type` = 'cdi-cdd' AND `Wish`.`services` REGEXP '(Radiothérapie)' INNER JOIN `Candidates` AS `Wish->Candidate` ON `Wish`.`candidate_id` = `Wish->Candidate`.`id` INNER JOIN `Users` AS `Wish->Candidate-
>User` ON `Wish->Candidate`.`user_id` = `Wish->Candidate->User`.`id` LEFT OUTER JOIN `Experiences` AS `Wish->Candidate->experiences` ON `Wish->Candidate`.`id` = `Wish->Candidate->experiences`.`candidate_id` LEFT OUTER JOIN `CandidateDocuments` AS `Wish->Candidate->
documents` ON `Wish->Candidate`.`id` = `Wish->Candidate->documents`.`candidate_id` LEFT OUTER JOIN `CandidateFormations` AS `Wish->Candidate->formations` ON `Wish->Candidate`.`id` = `Wish->Candidate->formations`.`candidate_id` LEFT OUTER JOIN `Establishments` AS `E
stablishment` ON `Application`.`es_id` = `Establishment`.`id` LEFT OUTER JOIN `FavoriteCandidates` AS `Establishment->FavoriteCandidates` ON `Establishment`.`id` = `Establishment->FavoriteCandidates`.`es_id` AND `Wish->Candidate`.`id` = `Establishment->FavoriteCand
idates`.`candidate_id` LEFT OUTER JOIN `Users` AS `Establishment->FavoriteCandidates->User` ON `Establishment->FavoriteCandidates`.`added_by` = `Establishment->FavoriteCandidates->User`.`id` LEFT OUTER JOIN `ArchivedCandidates` AS `Establishment->ArchivedCandidates
` ON `Establishment`.`id` = `Establishment->ArchivedCandidates`.`es_id` AND `Wish->Candidate`.`id` = `Establishment->ArchivedCandidates`.`candidate_id` WHERE `Application`.`es_id` IN ('5') GROUP BY `Wish->Candidate`.`id`;
```

(Specifically this part of generated request : 
```sql
INNER JOIN `Wishes` AS `Wish` ON `Application`.`wish_id` = `Wish`.`id` AND lower(`posts`) LIKE '%asd%' AND `Wish`.`contract_type` = 'cdi-cdd' AND `Wish`.`services` REGEXP '\"(Radiothérapie)\"'
```

instead of 
```sql
INNER JOIN `Wishes` AS `Wish` ON `Application`.`wish_id` = `Wish`.`id` AND lower(`posts`) LIKE '%asd%' AND `Wish`.`contract_type` = 'cdi-cdd' AND `Wish`.`services` REGEXP '(Radiothérapie)'
```
)

When I do my test directly in mysql command line, `REGEXP '(Radiothérapie)'` works and give me awaited results but `REGEXP '\"(Radiothérapie)\"'` not.

**Comment by SimonSchick** [comment]
This is not a minimal repro, please post a *self contained* example.

**Comment by SimonSchick** [comment]
Fyi I tried to replicate this:

```
'use strict';

const S = require('./');
const i = new S('mysql://root:pw@localhost/test123');
class User extends S.Model {}
User.init({ id: { type: S.INTEGER, primaryKey: true }, login: S.STRING }, { tableName: 'users', timestamps: false, sequelize: i });
class Project extends S.Model {}
Project.init({ id: { type: S.INTEGER, primaryKey: true }, name: S.STRING }, { tableName: 'projects', timestamps: false, sequelize: i });

User.hasOne(Project);

i.sync()
  .then(() =>
    User.findAll({
      logging: true,
      include: [{
        model: Project,
        where: {
          name: {
            [S.Op.regexp]: `(${['asd']})`,
          }
        }
      }],
    })
  )
```

Result:
```
SELECT `User`.`id`, `User`.`login`, `Project`.`id` AS `Project.id`, `Project`.`name` AS `Project.name`, `Project`.`UserId` AS `Project.UserId` FROM `users` AS `User` INNER JOIN `projects` AS `Project` ON `User`.`id` = `Project`.`UserId` AND `Project`.`name` REGEXP '(asd)';
```

I assume that your `filterQuery.services[0]` is not a string but instead something that gets stringified and delimited with quotes.

**Comment by Sorok-Dva** [comment]
Thanks for your answer, here is minimal repro. (I hope, I'm sorry but I'm not sure about what you are waiting exactly)

```javascript
// Model Declaration
'use strict';
module.exports = (sequelize, DataTypes) => {
  const Wish = sequelize.define('Wish', {
    candidate_id: DataTypes.INTEGER,
    name: DataTypes.STRING,
    contract_type: DataTypes.STRING,
    posts: {
      type: DataTypes.JSON,
      get() {
        return JSON.parse(this.getDataValue('posts'))
      },
      set(data) {
        this.setDataValue('posts', JSON.stringify(data));
      }
    },
    services: {
      type: DataTypes.JSON
    },
    full_time: DataTypes.BOOLEAN,
    part_time: DataTypes.BOOLEAN,
    day_time: DataTypes.BOOLEAN,
    night_time: DataTypes.BOOLEAN,
    liberal_cabinets: DataTypes.BOOLEAN,
    availability: DataTypes.JSON,
    start: DataTypes.DATE,
    end: DataTypes.DATE,
    status: DataTypes.STRING,
    lat: DataTypes.STRING,
    lon: DataTypes.STRING,
    geolocation: DataTypes.BOOLEAN,
    custom_address: DataTypes.TEXT,
    es_count: DataTypes.INTEGER
  }, {});
  Wish.associate = function (models) {
    Wish.hasOne(models.Candidate, {
      foreignKey: 'id',
      sourceKey: 'candidate_id',
      onDelete: 'CASCADE'
    });
  };
  return Wish;
};

// Query

return Models.Wish.findAll({
    where: {
      services: {
        [Op.regexp]: `(test)`
      }
    }
  }).then(data => {
    console.log(data);
  })
```

That still generate me 
```sql 
SELECT `id`, `candidate_id`, `name`, `contract_type`, `posts`, `services`, `full_time`, `part_time`, `day_time`, `night_time`, `liberal_cabinets`, `availability`, `start`, `end`, `status`, `lat`, `lon`, `geolocation`, `custom_address`, `es_count`, `createdAt`, `updatedAt` FROM `Wishes` AS `Wish` WHERE `Wish`.`services` REGEXP '\"(test)\"';
```
 instead of 

```sql
SELECT `id`, `candidate_id`, `name`, `contract_type`, `posts`, `services`, `full_time`, `part_time`, `day_time`, `night_time`, `liberal_cabinets`, `availability`, `start`, `end`, `status`, `lat`, `lon`, `geolocation`, `custom_address`, `es_count`, `createdAt`, `updatedAt` FROM `Wishes` AS `Wish` WHERE `Wish`.`services` REGEXP '(test)';
```

And I tried it a differents way, so in this case I do not use `filterQuery.services[0]` but it still creates me some backslashes and double quote. And for information, in the case when I use `filterQuery.services[0]`, it was  indeed typeof string and when I console.log query object I'd seen the right query with not backslashes and quotes.

**Comment by SimonSchick** [comment]
That's still not a minimal repro but I see the issue now, the column is marked as JSON and therefore stringification/escaping is applied here.

**Comment by SimonSchick** [comment]
For now try using `literal(YOUR_VALUE)` while I look into a bug fix.

**Comment by Sorok-Dva** [comment]
Thanks for you quick answer and your temp fix.

So now I use 

```javascript
return Models.Wish.findAll({
    where: {
      services: {
        [Op.regexp]: Sequelize.literal(`'(${filterQuery.services[0]})'`)
      }
    }
  });
```
that works like a charm !

**Comment by amarjit-singh** [comment]
> For now try using `literal(YOUR_VALUE)` while I look into a bug fix.

But I think It can open doors for SQL Injection.

**Comment by papb** [comment]
@amarjit-singh Indeed, that is why it was temporary, a proper fix was already done in #10615 :)
